app/parse.py

```diff
@@ -1,4 +1,13 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, fields, astuple
+
+from bs4 import BeautifulSoup
+
+import csv
+
+import requests
+
+BASE_URL = "https://quotes.toscrape.com/"
+PAGE_URL = f"{BASE_URL}page/{{}}"
```

Comment:
```suggestion
PAGE_URL = f"{BASE_URL}page/{page_number}"
```

Reply:
and how to define page_number? I  should define PAGE_URL in get_page_content function?

Reply:
with using PAGE_URL.format(page_number)

Reply:
![image](https://user-images.githubusercontent.com/107995555/194258753-7fe64561-56e4-48d0-bcc1-efd2e2250e43.png)
When I use variable page_number I get error

Reply:
PAGE_URL = BASE_URL + "page/{page_number}"